interface/expand_contract_js.php

```diff
@@ -0,0 +1,41 @@
+<?php
+  /**
+ * expand contract jquery script
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+// ensure that $user_settings_php_path, $arr_files_php variables are set in the script calling this script
+?>
+$( document ).ready(function() {
+    $('.expand_contract').click(function() {
+        var elementTitle = $(this).prop('title');
+        var contractTitle = '<?php echo xla('Click to Contract and set to henceforth open in Centered mode'); ?>';
+        var expandTitle = '<?php echo xla('Click to Expand and set to henceforth open in Expanded mode'); ?>';
+        var arrFiles = <?php echo json_encode($arr_files_php) ?>;
```

Comment:
Since $arr_files_php variables are hard-coded in the script and based on filenames, we are ok in not html escaping them (ie. ignore that issue and get this thing done :) )